src/templates/layouts/vamcSystem/index.tsx

```diff
@@ -0,0 +1,172 @@
+import { VamcSystem as FormattedVamcSystem } from '@/types/formatted/vamcSystem'
+import { MediaImage } from '@/templates/common/mediaImage'
+import { ContentFooter } from '@/templates/common/contentFooter'
+import { LovellStaticPropsResource } from '@/lib/drupal/lovell/types'
+import { useEffect } from 'react'
+import { SideNavMenu } from '@/types/formatted/sideNav'
+// import { LovellSwitcher } from '@/templates/components/lovellSwitcher'
+// import { TopTasks } from '@/templates/components/topTasks'
+// import { FacilityListing } from '@/templates/components/facilityListing'
+// import { MainButtons } from '@/templates/components/mainButtons'
+// import { ListOfLinkTeasers } from '@/templates/components/listOfLinkTeasers'
+// import { NewsStoryTeaser } from '@/templates/components/newsStoryTeaser'
+// import { EventTeaser } from '@/templates/components/eventTeaser'
+// import { SocialLinks } from '@/templates/common/socialLinks'
+
+const LOVELL_TRICARE_ADMINISTRATION_ID = 1039
+const LOVELL_VA_ADMINISTRATION_ID = 1040
+
+// Allows additions to window object without overwriting global type
+interface customWindow extends Window {
+  sideNav?: SideNavMenu
+}
+declare const window: customWindow
+
+export function VamcSystem({
+  title,
+  introText,
+  image,
+  administration,
+  fieldRelatedLinks,
+  path,
+  menu,
+  // vamcEhrSystem,
+  // mainFacilities,
+  // newsStoryTeasersFeatured,
+  // eventTeasersFeatured,
+  // eventTeasersAll,
+  // lovellVariant,
+  // lovellSwitchPath,
+}: LovellStaticPropsResource<FormattedVamcSystem>) {
+  // Populate the side nav data for the side nav widget to fill in
+  // Note: The side nav widget is in a separate app in the static-pages bundle
+  useEffect(() => {
+    window.sideNav = menu
+  })
+
+  const imageStyle = '7_2_medium_thumbnail'
+  const hasValidImage = image?.links?.[imageStyle]?.href
+
+  return (
+    <div className="va-l-detail-page va-facility-page">
+      <div className="usa-grid usa-grid-full">
+        {/* Nav data fille in by a separate script from `window.sideNav` */}
+        <nav aria-label="secondary" data-widget-type="side-nav" />
+        {/* Main page content */}
+        <div className="usa-width-three-fourths">
+          <article className="usa-content va-l-facility-detail vads-u-padding-bottom--0">
+            {/* <LovellSwitcher
+              currentVariant={lovellVariant}
+              switchPath={lovellSwitchPath}
+            /> */}
+            {title && <h1>{title}</h1>}
+            {hasValidImage && (
+              <div className="duotone darken lighten medium-screen:vads-u-margin-bottom--0p5">
+                <MediaImage {...image} imageStyle={imageStyle} />
```

Comment:
@laflannery, it looks like the `alt` text is coming from the Drupal data for the image. We use this `<MediaImage>` component and pattern in several different components/pages, which means we might need to watch out for that in other places. I'll go ahead and override the `alt` attribute that we get from Drupal with `""` here.

Reply:
Oh actually, I lied; I can't just change it here. I would also need to [updated this logic](https://github.com/department-of-veterans-affairs/next-build/blob/pwolfert/vamc-system/src/templates/common/mediaImage/index.tsx#L67) so we explicitly check for an `alt` value of `null` or `undefined` in the `MediaImage` component and allow for explicitly setting `alt=""`. It makes sense that we wouldn't want to allow the attribute to be completely omitted from the element, but we should start allowing for the case where it's only presentational.

Reply:
CC @cvalarida